File: app/api/rooms/rooms.js
'use strict';

// List active rooms with user counts and active call counts
const url = 'http://localhost:8000/api/v1/rooms';

const authorization = 'call_me_api_key_secret';

fetch(url, {
    method: 'GET',
    headers: {
        Authorization: authorization,
        'Content-Type': 'application/json',
    },
})
    .then((response) => response.json())
    .then((data) => console.log(data))
    .catch((error) => console.error('Error:', error));

File: app/api/rooms/rooms.py
import requests # pip3 install requests

# List active rooms with user counts and active call counts
url = "http://localhost:8000/api/v1/rooms"

authorization = "call_me_api_key_secret"

headers = {
    'Authorization': authorization,
    'Content-Type': 'application/json'
}

response = requests.get(url, headers=headers)

print(response.json())
